src/Orleans/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": "1.0.0-*",
   "dependencies": {
-    "Newtonsoft.Json": "7.0.1"
+    "Newtonsoft.Json": "9.0.1"
```

Comment:
I was going to leave it alone but wasn't sure if you wanted master on the latest or not.

I can revert this stuff but I'm now on vacation until Tuesday.  Will fix then.